### ISSUE TITLE
Pinta 1.7 was released few days back can new version be packaged

### ISSUE THREAD

**Issue by igor2x** [flathub/com.github.PintaProject.Pinta#12]
Pinta 1.7. was released few days back, see Release Notes: https://www.pinta-project.com/releases/1-7
Can Pinta's flatpack developers please package new version.

According to my tests, Pinta 1.7 is very stable using with Mono 6.x https://www.mono-project.com/download/stable/
but has some nasty freezes with older version of Mono.

**Comment by darkdragon-001** [comment]
I tested it with master a few weeks prior to release in #10. Together with Mono 6.x in flathub/flathub#1647, this should work like a charm.

**Comment by ghost** [comment]
Any update on this? When is the newer version going to be released?

**Comment by darkdragon-001** [comment]
@grande-mephistopheles Can you help testing #13 ? Just upgrade and tell us if you find any issues or if anything goes smooth.

**Comment by ghost** [comment]
@darkdragon-001 I have installed from the pull and seem to be working fine.
OS: Linux Mint 20 Cinnamon
```
flatpak install --user https://dl.flathub.org/build-repo/26223/com.github.PintaProject.Pinta.flatpakref
The application com.github.PintaProject.Pinta depends on runtimes from:
  https://dl.flathub.org/repo/
Configure this as new remote 'flathub' [Y/n]: y

com.github.PintaProject.Pinta permissions:
    ipc                   network              x11      dri
    file access [1]       dbus access [2]

    [1] /run/media, xdg-desktop, xdg-documents, xdg-download, xdg-pictures,
        xdg-public-share, xdg-run/dconf, xdg-run/gvfs:ro, xdg-templates,
        ~/.config/dconf:ro
    [2] ca.desrt.dconf, org.gtk.vfs, org.gtk.vfs.*


        ID                                   Branch Op Remote       Download
 1. [✓] com.github.PintaProject.Pinta.Locale test   i  pinta-origin  8.2 kB / 6.9 MB
 2. [✓] com.github.PintaProject.Pinta        test   i  pinta-origin 27.1 MB / 33.8 MB

Installation complete.
```
![pinta1](https://user-images.githubusercontent.com/58479925/92928064-1d91f880-f42e-11ea-96aa-64075c82a851.png)

Good to go. :+1:  Please update the system repo so I can remove this one

**Comment by igor2x** [comment]
Good job!

I have tested flatpak's Pinta on Ubuntu 20.04 and it works great. The only small issue is it does not integrate so well into Ubuntu environment like snap's Pinta version.

Left image is Pinta installed from snap package:
`snap install pinta-james-carroll`

Right image is Pinta installed from flatpak package:
`flatpak install --user https://dl.flathub.org/build-repo/26223/com.github.PintaProject.Pinta.flatpakref`

![snap_vs_flatpak](https://user-images.githubusercontent.com/10895030/92936163-0fea6c00-f44a-11ea-9095-c485baf95def.png)
Snap Pinta (left) is more Ubuntu style (Ubuntu uses orange colour) and it looks more integrated into Ubuntu.
Flatpak Pinta (right) is more Fedora style (Fedora uses blue colour).

This is the only flatpak program I have tested, so don't know if this is Pinta specific problem or how in general flatpak applications are displayed in Ubuntu.

P.S. This is probably very small issue and first version of Pinta flatpack 1.7 can be released. This is way way way more stable then 5 years old Pinta 1.6 that is full of crashes.
